samples/grids/grid/remote-paging-grid/src/index.tsx

```diff
@@ -1,19 +1,13 @@
 import React, { useEffect, useRef, useState } from "react";
-
 import ReactDOM from "react-dom/client";
-import "./index.css";
-
-import { IgrGrid, IgrPaginator, IgrGridModule, GridPagingMode } from "igniteui-react-grids";
+import { IgrGrid, IgrPaginator, GridPagingMode } from "igniteui-react-grids";
 import { IgrColumn } from "igniteui-react-grids";
-
 import "igniteui-react-grids/grids/combined";
 import "igniteui-react-grids/grids/themes/light/bootstrap.css";
 import { RemoteService } from "./RemotePagingService";
 import { CustomersWithPageResponseModel } from "./CustomersWithPageResponseModel";
 import { IgrNumberEventArgs } from "igniteui-react";
 
-IgrGridModule.register();
-
 export default function App() {
   const grid = useRef<IgrGrid>(null);
   const paginator = useRef<IgrPaginator>(null);
```

Comment:
Also in general, refrain from manipulating state through refs - if you're seeing `useRef` in the sample and it's not being used to call methods - suspect its usage entirely and see if it can be replaced with state and bindings instead.
I.e.:
paginator  totalRecords _can and should be_ state that's bound to the paginator, that's the expected usage pattern and it should work